src/sentry/seer/assisted_query/issues_tools.py

```diff
@@ -523,3 +523,57 @@ def execute_issues_query(
     )
 
     return resp.data
+
+
+def get_issues_stats(
+    *,
+    org_id: int,
+    issue_ids: list[str],
+    project_ids: list[int],
+    query: str,
+    stats_period: str = "24h",
```

Comment:
```suggestion
    stats_period: str = "7d",
```
stays consistent with issues list call